UnityProject/Assets/Scripts/Closets/ClosetControl.cs

```diff
@@ -194,26 +208,28 @@ private void SetItems(bool open)
 
         private void SetItemsAliveState(bool on)
         {
-			if (!on)
-			{
-				heldItems = matrix.Get<ObjectBehaviour>(registerTile.Position, ObjectType.Item);
-			}
+            if (!on)
+            {
+                heldItems = matrix.Get<ObjectBehaviour>(registerTile.Position, ObjectType.Item);
+            }
             foreach (var item in heldItems)
             {
-                if (on) {
-					item.transform.position = transform.position;
-				}
-
+                if (on)
+                {
+                    item.transform.position = transform.position;
+                }
+
                 item.visibleState = on;
             }
         }
 
         private void SetPlayersAliveState(bool on)
         {
             if (!on)
-			{
-				heldPlayers = matrix.Get<ObjectBehaviour>(registerTile.Position, ObjectType.Player);
-			}
+            {
+                Debug.Log("ran");

```

Comment:
Please don't add debug.log, unless at the very least it's clear from the log what has gone wrong/good.